Sources/Starknet/Data/Transaction/TransactionWrapper.swift

```diff
@@ -11,7 +11,8 @@ enum TransactionWrapper: Decodable {
     case invokeV0(StarknetInvokeTransactionV0)
     case deployAccount(StarknetDeployAccountTransaction)
     case deploy(StarknetDeployTransaction)
-    case declare(StarknetDeclareTransaction)
+    case declareLegacy(StarknetDeclareTransactionLegacy)
```

Comment:
Maybe let's use one pattern for all versioned methods, and name it declareV1?

Reply:
I named this way on purpose, as this declareLagacy can be both v0 and v1

Reply:
I think declare 0 is already deprecated and removed, but let's double check